features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/db2.sql

```diff
@@ -754,6 +754,28 @@ CREATE TRIGGER IDP_ROLE_TRIG NO CASCADE
                     SET (NEW.ID) = (NEXTVAL FOR IDP_ROLE_SEQ);
                 END
 /
+CREATE TABLE IDP_GROUP (
+			ID INTEGER NOT NULL,
+			IDP_ID INTEGER NOT NULL,
+			TENANT_ID INTEGER NOT NULL,
+			GROUP_NAME VARCHAR(255) NOT NULL,
```

Comment:
these not null constraint is missing in other db scripts. pls check

Reply:
Added in other scripts as well with b8de1fc6d7537d6ad24585c99c34aca0ca442d92